### DISCUSSION TITLE
export default defineUserConfig使用插件

### DISCUSSION THREAD

**Discussion by zybzbk** [vuepress-theme-hope/vuepress-theme-hope#4229] [Q&A] [answered]
我在export default defineUserConfig中，该怎么使用extendsMarkdown来使用vuepress-plugin-md-enhance插件

**Comment by Mister-Hope** [comment, marked as answer]
vuepress plugin is totally a different thing with markdown-it plugin,  It seems you are regarding them as the same thing.

For markdown plugins, try things like  https://mdit-plugins.github.io/,  You should use most of them successfully if you want them

**Reply by zybzbk** [reply]
> vuepress 插件与 markdown-it 插件完全是两回事，似乎你把它们视为同一件事。
> 
> 对于 markdown 插件，可以尝试 https://mdit-plugins.github.io/之类的插件，如果你需要，你可以成功使用其中的大部分插件

那我应该如何在export default defineUserConfig设置中使用vuepress 插件
